### ISSUE TITLE
Question about the direct FT

### ISSUE THREAD

**Issue by yangzhangok** [Wakals/CoVT#19]
<img width="435" height="249" alt="Image" src="https://github.com/user-attachments/assets/b691dc7e-bd30-49d8-822f-c377f7975ce5" />

We just tried to reproduce the result in the ablation experiment(direct FT) but failed. May I ask what the setting of 0 token is?(direct FT)

**Comment by Wakals** [comment]
Hi, it is without using any visual tokens, just use the same dataset to finetune Qwen2.5-VL with the same configuration.
Best,

**Comment by yangzhangok** [comment]
Yes,we use the setting of stage3 ( learning rate,dataset and lora config the same with the paper) and trained for 8k step,but the score on CVBench is almost 80.

**Comment by VoyageWang** [comment]
@yangzhangok  Hi! Have you successfully trained the model with a visual chain of thought? Can I have a communication with you?
